ami.tf

```diff
@@ -32,6 +25,8 @@ data "aws_ami" "openvpn" {
     values = ["ebs"]
   }
 
-  owners      = ["self"]
+  owners = [
```

Comment:
I didn't realize this is an array.  This raises interesting questions / solutions.